src/qz/ui/AboutDialog.java

```diff
@@ -1,150 +1,200 @@
+
+        productInfoBlock = new JEditorPane("text/html", "<html>" + styleBlock + infoBlocks[1] + "</html>");
+        productInfoBlock.setMaximumSize(new Dimension(320, 200));
+        productInfoBlock.setEditable(false);
+        productInfoBlock.setOpaque(false);
+        productInfoBlock.addHyperlinkListener(linkListener);
+        productInfoBlock.putClientProperty(JEditorPane.HONOR_DISPLAY_PROPERTIES, true);
+        ((HTMLEditorKit)productInfoBlock.getEditorKit()).setStyleSheet(((HTMLEditorKit)versionInfoBlock.getEditorKit()).getStyleSheet());
+
+        JPanel infoBlock = new JPanel();
+        infoBlock.setPreferredSize(new Dimension(320, 300));
+        infoBlock.setLayout(new BoxLayout(infoBlock, BoxLayout.Y_AXIS));
+        infoBlock.add(versionInfoBlock);
+        Box btnBox = Box.createHorizontalBox();
+        btnBox.add(updateButton);
+        btnBox.add(Box.createHorizontalGlue());
+        infoBlock.add(btnBox);
+        infoBlock.add(productInfoBlock);
+
+        JPanel aboutPanel = new JPanel(new FlowLayout(FlowLayout.CENTER));
+        aboutPanel.add(new JLabel(getIcon(IconCache.Icon.LOGO_ICON)));
+        aboutPanel.add(infoBlock);
+
+
+        //override font to remove underline for these links
+        Font lblFont = new Font(null, Font.PLAIN, 12);
+        Map<TextAttribute,Object> attributes = new HashMap<>(lblFont.getAttributes());
+        attributes.remove(TextAttribute.UNDERLINE);
+        lblFont = lblFont.deriveFont(attributes);
+
+        LinkLabel lblLicensing = new LinkLabel("Licensing Information", Constants.ABOUT_URL + "/licensing ");
+        lblLicensing.setFont(lblFont);
+        LinkLabel lblSupport = new LinkLabel("Support Information", Constants.ABOUT_URL + "/support");
+        lblSupport.setFont(lblFont);
+        LinkLabel lblPrivacy = new LinkLabel("Privacy Policy", Constants.ABOUT_URL + "/privacy");
+        lblPrivacy.setFont(lblFont);
+
+        JPanel supportPanel = new JPanel(new FlowLayout(FlowLayout.CENTER, 80, 10));
+        supportPanel.add(lblLicensing);
+        supportPanel.add(lblSupport);
+        supportPanel.add(lblPrivacy);
+
+
+        JPanel panel = new JPanel();
+        panel.setLayout(new BoxLayout(panel, BoxLayout.PAGE_AXIS));
+        panel.add(aboutPanel);
+        panel.add(new JToolBar.Separator());
+        panel.add(supportPanel);
+
+
+        setContent(panel, true);
     }
 
-    public JButton addPanelButton(JMenuItem menuItem) {
-        JButton button = addPanelButton(menuItem.getText(), menuItem.getIcon(), menuItem.getMnemonic());
-        button.addActionListener(menuItem.getActionListeners()[0]);
-        return button;
+    @Override
+    public void refresh() {
+        super.refresh();
+
+        //ensure jeditorpane link colors are updated..
+        HTMLEditorKit kit = (HTMLEditorKit)versionInfoBlock.getEditorKit();
+        StyleSheet ss = kit.getStyleSheet();
+
+        //FIXME - broken changing lite -> dark
```

Comment:
Per our PM, these may be better handled as labels so that the theming behaves properly.